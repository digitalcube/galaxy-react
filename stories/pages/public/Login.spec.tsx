import { render } from '@testing-library/react';
import React from 'react';
import { Default as Login, Inprogress, Failure } from './Login.stories';

describe('Login', () => {
  it('to match snapshot(default)', () => {
    const { asFragment } = render(<Login {...(Login.args as any)} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('to match snapshot (inprogress)', () => {
    const { asFragment } = render(<Inprogress {...(Inprogress.args as any)} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('to match snapshot (failure)', () => {
    const { asFragment } = render(<Failure {...(Failure.args as any)} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
