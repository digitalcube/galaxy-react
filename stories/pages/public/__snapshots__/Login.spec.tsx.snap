// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login to match snapshot (failure) 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-column min-vh-100 shifter-dashboard-login"
  >
    <div
      class="mt-4 mb-4 flex-grow-1 d-flex align-items-center justify-content-center"
    >
      <div
        class="mx-auto d-flex flex-column flex-md-row align-items-center justify-content-center justify-content-md-between login-column"
      >
        <div
          class="login px-4 py-4 rounded d-flex flex-column justify-content-center mx-3 error"
        >
          <form
            novalidate=""
          >
            <div
              class="mb-4 pb-2 login-header"
            >
              <div
                class="mb-4 text-center login-icon"
              >
                <img
                  alt="shifter"
                  height="46"
                  width="36"
                />
              </div>
              <h1
                class="mb-0 font-weight-bold text-center login-title"
              >
                Log In
              </h1>
              <div
                class="mt-2 font-weight-bold text-center login-switch-way"
              >
                or 
                <a
                  href="/signup"
                  to="/signup"
                >
                  sign up
                </a>
              </div>
            </div>
            <div
              class="alert alert-danger mb-4 p-0 border-0 d-flex c-alert"
              role="alert"
            >
              <div
                class="alert-icon px-1 d-flex align-items-center"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
                    />
                  </svg>
                </i>
              </div>
              <div
                class="alert-body px-3 py-2 flex-grow-1"
              >
                <p
                  class="mb-0"
                >
                  We found some errors with your login info. Please correct these issues to continue
                </p>
              </div>
            </div>
            <div
              class="mb-4 form-group"
            >
              <label
                class="font-weight-bold"
                for="username"
              >
                Username or email
              </label>
              <div
                class="reverse-icon input-group"
              >
                <input
                  class="is-invalid form-control"
                  id="username"
                  placeholder="Username or email"
                  type="text"
                  value=""
                />
                <div
                  class="invalid-feedback"
                >
                  We don’t recognize that email
                </div>
                <div
                  class="input-group-append"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      />
                    </svg>
                  </i>
                </div>
              </div>
            </div>
            <div
              class="mb-4 form-group"
            >
              <label
                class="font-weight-bold"
                for="password"
              >
                Password
              </label>
              <div
                class="reverse-icon input-group"
              >
                <input
                  class="is-invalid form-control"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <div
                  class="invalid-feedback"
                >
                  We don’t recognize that password
                </div>
                <div
                  class="input-group-append"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z"
                      />
                    </svg>
                  </i>
                </div>
                <div
                  class="input-group-eye password-lock"
                >
                  <i
                    aria-hidden="true"
                    class="lock"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 640 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M320 400c-75.85 0-137.25-58.71-142.9-133.11L72.2 185.82c-13.79 17.3-26.48 35.59-36.72 55.59a32.35 32.35 0 0 0 0 29.19C89.71 376.41 197.07 448 320 448c26.91 0 52.87-4 77.89-10.46L346 397.39a144.13 144.13 0 0 1-26 2.61zm313.82 58.1l-110.55-85.44a331.25 331.25 0 0 0 81.25-102.07 32.35 32.35 0 0 0 0-29.19C550.29 135.59 442.93 64 320 64a308.15 308.15 0 0 0-147.32 37.7L45.46 3.37A16 16 0 0 0 23 6.18L3.37 31.45A16 16 0 0 0 6.18 53.9l588.36 454.73a16 16 0 0 0 22.46-2.81l19.64-25.27a16 16 0 0 0-2.82-22.45zm-183.72-142l-39.3-30.38A94.75 94.75 0 0 0 416 256a94.76 94.76 0 0 0-121.31-92.21A47.65 47.65 0 0 1 304 192a46.64 46.64 0 0 1-1.54 10l-73.61-56.89A142.31 142.31 0 0 1 320 112a143.92 143.92 0 0 1 144 144c0 21.63-5.29 41.79-13.9 60.11z"
                      />
                    </svg>
                  </i>
                  <i
                    aria-hidden="true"
                    class="unlock"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 576 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
                      />
                    </svg>
                  </i>
                </div>
              </div>
            </div>
            <div
              class="form-check d-flex align-items-center colored-check smaller form-group"
            >
              <input
                class="screen-reader-text form-check-input"
                id="remember"
                label="Remember me"
                type="checkbox"
                value="false"
              />
              <label
                class="form-check-label"
                for="remember"
              >
                Remember me
              </label>
            </div>
            <button
              class="mt-4 btn-colored font-weight-bold btn btn-secondary btn-block disabled"
              disabled=""
              type="submit"
            >
              Log In
            </button>
            <div
              class="forget-password"
            >
              <a
                href="#"
                target="_blank"
              >
                Forgot Password?
              </a>
            </div>
          </form>
        </div>
        <div
          class="login-information mt-4 mt-md-0 mx-3"
        >
          <div
            class="login-information-icon text-center"
          >
            <img
              alt="login information"
              height="141"
              width="153"
            />
          </div>
          <h2
            class="login-information-title mb-0 font-weight-bold text-center"
          >
            Faster Build Times
          </h2>
          <div
            class="login-information-description mt-4 text-center"
          >
            Customize Shifter settings for up to 70% faster build times.
          </div>
          <div
            class="login-information-anchor font-weight-bold text-center"
          >
            <a
              href="#"
              to="#"
            >
              Learn about Shifter Media CDN
            </a>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="text-center"
    >
      <p
        class="m-0 common-footer-copyright"
      >
        <small
          class="d-flex align-items-center justify-content-center"
        >
          ©2021 Shifter | A Product by 
          <a
            href="https://www.digitalcube.jp/"
            rel="nofollow noopener"
            title="DigitalCube"
          >
            <img
              alt="digitalcube"
              class="d-block"
              height="15"
              width="77"
            />
          </a>
        </small>
      </p>
    </footer>
  </div>
</DocumentFragment>
`;

exports[`Login to match snapshot (inprogress) 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-column min-vh-100 shifter-dashboard-login"
  >
    <div
      class="mt-4 mb-4 flex-grow-1 d-flex align-items-center justify-content-center"
    >
      <div
        class="mx-auto d-flex flex-column flex-md-row align-items-center justify-content-center justify-content-md-between login-column"
      >
        <div
          class="login px-4 py-4 rounded d-flex flex-column justify-content-center mx-3 success"
        >
          <form
            novalidate=""
          >
            <div
              class="mb-4 pb-2 login-header"
            >
              <div
                class="mb-4 text-center login-icon"
              >
                <img
                  alt="shifter"
                  height="46"
                  width="36"
                />
              </div>
              <h1
                class="mb-0 font-weight-bold text-center login-title"
              >
                Log In
              </h1>
              <div
                class="mt-2 font-weight-bold text-center login-switch-way"
              >
                or 
                <a
                  href="/signup"
                  to="/signup"
                >
                  sign up
                </a>
              </div>
            </div>
            <div
              class="mb-4 form-group"
            >
              <label
                class="font-weight-bold"
                for="username"
              >
                Username or email
              </label>
              <div
                class="reverse-icon input-group"
              >
                <input
                  class="form-control"
                  id="username"
                  placeholder="Username or email"
                  type="text"
                  value=""
                />
                <div
                  class="input-group-append"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      />
                    </svg>
                  </i>
                </div>
              </div>
            </div>
            <div
              class="mb-4 form-group"
            >
              <label
                class="font-weight-bold"
                for="password"
              >
                Password
              </label>
              <div
                class="reverse-icon input-group"
              >
                <input
                  class="form-control"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <div
                  class="input-group-append"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z"
                      />
                    </svg>
                  </i>
                </div>
                <div
                  class="input-group-eye password-lock"
                >
                  <i
                    aria-hidden="true"
                    class="lock"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 640 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M320 400c-75.85 0-137.25-58.71-142.9-133.11L72.2 185.82c-13.79 17.3-26.48 35.59-36.72 55.59a32.35 32.35 0 0 0 0 29.19C89.71 376.41 197.07 448 320 448c26.91 0 52.87-4 77.89-10.46L346 397.39a144.13 144.13 0 0 1-26 2.61zm313.82 58.1l-110.55-85.44a331.25 331.25 0 0 0 81.25-102.07 32.35 32.35 0 0 0 0-29.19C550.29 135.59 442.93 64 320 64a308.15 308.15 0 0 0-147.32 37.7L45.46 3.37A16 16 0 0 0 23 6.18L3.37 31.45A16 16 0 0 0 6.18 53.9l588.36 454.73a16 16 0 0 0 22.46-2.81l19.64-25.27a16 16 0 0 0-2.82-22.45zm-183.72-142l-39.3-30.38A94.75 94.75 0 0 0 416 256a94.76 94.76 0 0 0-121.31-92.21A47.65 47.65 0 0 1 304 192a46.64 46.64 0 0 1-1.54 10l-73.61-56.89A142.31 142.31 0 0 1 320 112a143.92 143.92 0 0 1 144 144c0 21.63-5.29 41.79-13.9 60.11z"
                      />
                    </svg>
                  </i>
                  <i
                    aria-hidden="true"
                    class="unlock"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 576 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
                      />
                    </svg>
                  </i>
                </div>
              </div>
            </div>
            <div
              class="form-check d-flex align-items-center colored-check smaller form-group"
            >
              <input
                class="screen-reader-text form-check-input"
                id="remember"
                label="Remember me"
                type="checkbox"
                value="false"
              />
              <label
                class="form-check-label"
                for="remember"
              >
                Remember me
              </label>
            </div>
            <button
              class="mt-4 btn-colored font-weight-bold btn btn-secondary btn-block disabled"
              disabled=""
              type="submit"
            >
              Log In
            </button>
            <div
              class="forget-password"
            >
              <a
                href="#"
                target="_blank"
              >
                Forgot Password?
              </a>
            </div>
          </form>
        </div>
        <div
          class="login-information mt-4 mt-md-0 mx-3"
        >
          <div
            class="login-information-icon text-center"
          >
            <img
              alt="login information"
              height="141"
              width="153"
            />
          </div>
          <h2
            class="login-information-title mb-0 font-weight-bold text-center"
          >
            Faster Build Times
          </h2>
          <div
            class="login-information-description mt-4 text-center"
          >
            Customize Shifter settings for up to 70% faster build times.
          </div>
          <div
            class="login-information-anchor font-weight-bold text-center"
          >
            <a
              href="#"
              to="#"
            >
              Learn about Shifter Media CDN
            </a>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="text-center"
    >
      <p
        class="m-0 common-footer-copyright"
      >
        <small
          class="d-flex align-items-center justify-content-center"
        >
          ©2021 Shifter | A Product by 
          <a
            href="https://www.digitalcube.jp/"
            rel="nofollow noopener"
            title="DigitalCube"
          >
            <img
              alt="digitalcube"
              class="d-block"
              height="15"
              width="77"
            />
          </a>
        </small>
      </p>
    </footer>
  </div>
</DocumentFragment>
`;

exports[`Login to match snapshot(default) 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-column min-vh-100 shifter-dashboard-login"
  >
    <div
      class="mt-4 mb-4 flex-grow-1 d-flex align-items-center justify-content-center"
    >
      <div
        class="mx-auto d-flex flex-column flex-md-row align-items-center justify-content-center justify-content-md-between login-column"
      >
        <div
          class="login px-4 py-4 rounded d-flex flex-column justify-content-center mx-3"
        >
          <form
            novalidate=""
          >
            <div
              class="mb-4 pb-2 login-header"
            >
              <div
                class="mb-4 text-center login-icon"
              >
                <img
                  alt="shifter"
                  height="46"
                  width="36"
                />
              </div>
              <h1
                class="mb-0 font-weight-bold text-center login-title"
              >
                Log In
              </h1>
              <div
                class="mt-2 font-weight-bold text-center login-switch-way"
              >
                or 
                <a
                  href="/signup"
                  to="/signup"
                >
                  sign up
                </a>
              </div>
            </div>
            <div
              class="mb-4 form-group"
            >
              <label
                class="font-weight-bold"
                for="username"
              >
                Username or email
              </label>
              <div
                class="reverse-icon input-group"
              >
                <input
                  class="form-control"
                  id="username"
                  placeholder="Username or email"
                  type="text"
                  value=""
                />
                <div
                  class="input-group-append"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      />
                    </svg>
                  </i>
                </div>
              </div>
            </div>
            <div
              class="mb-4 form-group"
            >
              <label
                class="font-weight-bold"
                for="password"
              >
                Password
              </label>
              <div
                class="reverse-icon input-group"
              >
                <input
                  class="form-control"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <div
                  class="input-group-append"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z"
                      />
                    </svg>
                  </i>
                </div>
                <div
                  class="input-group-eye password-lock"
                >
                  <i
                    aria-hidden="true"
                    class="lock"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 640 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M320 400c-75.85 0-137.25-58.71-142.9-133.11L72.2 185.82c-13.79 17.3-26.48 35.59-36.72 55.59a32.35 32.35 0 0 0 0 29.19C89.71 376.41 197.07 448 320 448c26.91 0 52.87-4 77.89-10.46L346 397.39a144.13 144.13 0 0 1-26 2.61zm313.82 58.1l-110.55-85.44a331.25 331.25 0 0 0 81.25-102.07 32.35 32.35 0 0 0 0-29.19C550.29 135.59 442.93 64 320 64a308.15 308.15 0 0 0-147.32 37.7L45.46 3.37A16 16 0 0 0 23 6.18L3.37 31.45A16 16 0 0 0 6.18 53.9l588.36 454.73a16 16 0 0 0 22.46-2.81l19.64-25.27a16 16 0 0 0-2.82-22.45zm-183.72-142l-39.3-30.38A94.75 94.75 0 0 0 416 256a94.76 94.76 0 0 0-121.31-92.21A47.65 47.65 0 0 1 304 192a46.64 46.64 0 0 1-1.54 10l-73.61-56.89A142.31 142.31 0 0 1 320 112a143.92 143.92 0 0 1 144 144c0 21.63-5.29 41.79-13.9 60.11z"
                      />
                    </svg>
                  </i>
                  <i
                    aria-hidden="true"
                    class="unlock"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 576 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
                      />
                    </svg>
                  </i>
                </div>
              </div>
            </div>
            <div
              class="form-check d-flex align-items-center colored-check smaller form-group"
            >
              <input
                class="screen-reader-text form-check-input"
                id="remember"
                label="Remember me"
                type="checkbox"
                value="false"
              />
              <label
                class="form-check-label"
                for="remember"
              >
                Remember me
              </label>
            </div>
            <button
              class="mt-4 btn-colored font-weight-bold btn btn-secondary btn-block disabled"
              disabled=""
              type="submit"
            >
              Log In
            </button>
            <div
              class="forget-password"
            >
              <a
                href="#"
                target="_blank"
              >
                Forgot Password?
              </a>
            </div>
          </form>
        </div>
        <div
          class="login-information mt-4 mt-md-0 mx-3"
        >
          <div
            class="login-information-icon text-center"
          >
            <img
              alt="login information"
              height="141"
              width="153"
            />
          </div>
          <h2
            class="login-information-title mb-0 font-weight-bold text-center"
          >
            Faster Build Times
          </h2>
          <div
            class="login-information-description mt-4 text-center"
          >
            Customize Shifter settings for up to 70% faster build times.
          </div>
          <div
            class="login-information-anchor font-weight-bold text-center"
          >
            <a
              href="#"
              to="#"
            >
              Learn about Shifter Media CDN
            </a>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="text-center"
    >
      <p
        class="m-0 common-footer-copyright"
      >
        <small
          class="d-flex align-items-center justify-content-center"
        >
          ©2021 Shifter | A Product by 
          <a
            href="https://www.digitalcube.jp/"
            rel="nofollow noopener"
            title="DigitalCube"
          >
            <img
              alt="digitalcube"
              class="d-block"
              height="15"
              width="77"
            />
          </a>
        </small>
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
